docs/user_guide/02_medrecord_intro.md

```diff
@@ -0,0 +1,239 @@
+
+```
+------------------------------------------------------
+Nodes Group Count Attribute Info                    
+------------------------------------------------------
+Diagnoses   1     ICD       Values: I.21             
+                  ICDv      min: 10                  
+                            max: 10                  
+                            mean: 10.00              
+Medications 2     Name      Values: Insulin, Wararin 
+Patients    3     Age       min: 64                  
+                            max: 74                  
+                            mean: 70.00              
+                  Loc       Values: GER, USA         
+                  Sex       Values: F, M             
+US-Patients 2     Age       min: 72                  
+                            max: 74                  
+                            mean: 73.00              
+                  Loc       Values: USA              
+                  Sex       Values: M                
+------------------------------------------------------
+```
+As shown, we have two groups of nodes - Patients and Medications - created when adding the nodes. Additionally, there’s a group called 'US-Patients' that we created. For each group of nodes, we can view their attributes along with a brief statistical summary, such as the minimum, maximum, and mean for numeric variables.
+
+We can do the same to get an overview over edges in our MedRecord by using the `print_edge_overview` method:
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 62
+---
+```
+
+```
+------------------------------------------------------
+Nodes Group Count Attribute Info                    
+------------------------------------------------------
+Diagnoses   1     ICD       Values: I.21             
+                  ICDv      min: 10                  
+                            max: 10                  
+                            mean: 10.00              
+Medications 2     Name      Values: Insulin, Wararin 
+Patients    3     Age       min: 64                  
+                            max: 74                  
+                            mean: 70.00              
+                  Loc       Values: GER, USA         
+                  Sex       Values: F, M             
+US-Patients 2     Age       min: 72                  
+                            max: 74                  
+                            mean: 73.00              
+                  Loc       Values: USA              
+                  Sex       Values: M                
+------------------------------------------------------
+```
+
+## Accessing Elements in a MedRecord
+
+Now that we have stored some structured data in our MedRecord, we might want to access certain elements of it. The main way to do this is by either selecting the data with their indices or via groups that they are in.
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 64-78
```

Comment:
Maybe better executing each cell separately? Or does it get too messy? This way no comments are needed

Reply:
yes, also thought so. The querying is also to be discussed in today's MM weekly